src/conf_mode/nat64.py

```diff
@@ -76,12 +81,9 @@ def get_config(config: Config | None = None) -> None:
         ):
             rules[num]["recreate"] = True
 
-    return nat64
-
-
-def verify(nat64) -> None:
     if not nat64:
-        # no need to verify the CLI as nat64 is going to be deactivated
+        unload_kmod(['jool'])
```

Comment:
I'm quite uneasy about system state changes in `verify()`. I'm inclined to think that `verify()` should never include any logic except config verification. Do you think it's feasible to move this to `apply()`?

Reply:
Agreed: though I consider the real problem with the logic of this script to be the need to modprobe in the `verify()` stage, no need to make it worse. Moved to `apply()`.